libraries/chain/include/graphene/chain/protocol/custom_authority.hpp

```diff
@@ -68,10 +67,21 @@ namespace graphene { namespace chain {
          uint32_t price_per_k_unit = 100; ///< units = valid seconds * items in auth * items in restrictions
       };
 
-      asset             fee;
-      account_id_type   account;
-      uint64_t          delta_units; // to calculate fee, it will be validated in evaluator
-                                     // Note: if start was in the past, when updating, used fee should be deducted
+      object_id_type                  custom_authority_to_update;
```

Comment:
Since we can pick a specific custom auth by id, we should consider setting the other attributes to `optional<>`. That way, we can simply re-enable an existing custom auth without having to provide the entire thing.
Also, we may only want to change `valid_to` but not `valid_from`.

Reply:
Thank you, will do.

Reply:
done